### PR TITLE
Improve website accessibility 

### PR DESCRIPTION
**Pull request by lbeckman314**
Ticks two of the boxes from #912 

- [x] Include labels for inputs (helps with screen reader accessibility even if visibly hidden)

- [x] Improve keyboard navigation (I suggest that tabbing moves between project titles and skips project tags & "up-for-grabs" label)

I'm still very new to pull requests, so any edits or recommendations on the commits are more than welcome. : )

### CODE REVIEW

File: _includes/scripts.html [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <script type="text/html" id="projects-panel-template">
   <div class="filters-panel cf">
-    <h5>Filter by name: </h5>
-    <select class="names-filter" multiple data-placeholder="Select a project..." >
+    <label for="project">Filter by name:</label>
+    <select class="names-filter" multiple data-placeholder="Select a project..." id="project">
     <% _.each(names, function(entry, key){ %>
```
Comment (allieholcombe) [inline _includes/scripts.html]: Almost identical to what I had in the works, only difference being that my ids were a smidgen more descriptive (`filter-by-name-input` etc.)

Comment (lbeckman314) [reply _includes/scripts.html]: You probably can't go wrong being more descriptive. In that case I would elect to use your improvements over mine!

Comment (ritwik12) [reply _includes/scripts.html]: @alexandraholcombe I think you haven't sent a PR, please send a PR.

Comment (allieholcombe) [reply _includes/scripts.html]: @ritwik12 @lbeckman314 This PR includes a few changes mine doesn't include.  I'll put that in this evening, but I recommend merging this first.

**Comment by shiftkey** [comment]
@lbeckman314 it looks like there's some conflicts on this PR, and I'm not sure of the resolution. Are you able to take a look?

**Comment by lbeckman314** [comment]
Absolutely. I will take a look.

**Comment by lbeckman314** [comment]
Fetched changes from upstream. Resolved and ready for review.

**Review by shiftkey — APPROVED** [review]
LGTM - thanks @lbeckman314!

**Pull request merged**